.env

```diff
@@ -5,8 +5,8 @@
 RUST_VERSION=stable
 
 # KAFKA
-# KAFKA_BROKERS="localhost:9093\,$LOCAL_HOST_IP:9094"
-KAFKA_BROKERS="localhost:9092"
+KAFKA_HOST="localhost\,$LOCAL_HOST_IP"
```

Comment:
Shouldn't we ignore .env files and not keep then in master branch?

Reply:
Good point I will remove from master